management_linux.go

```diff
@@ -73,9 +75,16 @@ func (m *Management) server(c net.Conn) { // {{{
 
 		if m.Conn.config.remote == "" {
 			log.Info("Management started in SERVER mode")
+			t := time.NewTicker( time.Second * 10)
```

Comment:
yes, this is mostly for status i am guessing. i will check if there is any side effect to it when testing on Vinay's setup

Reply:
hey :)

If I remember correctly (long time since I used this code) it asks the openvpn process for the current status once every second. The openvpn process then reports back a list of all connected clients.